chandra_aca/maude_decom.py

```diff
@@ -1135,9 +1137,9 @@ def get_aca_packets(
     if not blobs and not frames:
         frames = True
 
-    assert (blobs and not frames) or (
-        frames and not blobs
-    ), "Specify only one of 'frames' or blobs"
+    # check that only one of blobs or frames is set
+    if blobs and frames:
```

Comment:
This is not logically equivalent to the original.

Reply:
Thanks!

Reply:
I updated to something that may or may not be logically equivalent, but I think at least was the idea.  I'd first just updated to match what was printed, so changed the error text as well.